lib/tasks/update_printers.rake

```diff
@@ -6,12 +6,22 @@ def get_all_csv
   res = Net::HTTP.post_form(uri, 'textver' => 'CSV export')
   xml = Nokogiri::XML(res.body)
   text = xml.css('pre').children.text.strip
-  CSV.parse(text, {
+  csv = CSV.parse(text, {
     col_sep: "\t",
     header_converters: [lambda {|f| f.strip}, :symbol],
     converters: lambda {|f| f ? f.strip : nil },
     headers: true
   })
+  check_duplex csv
+end
+
+def check_duplex(csv)
+  csv.map do |p|
+    sizes = p[:size].split
+    p[:size] = (sizes - %w(Duplex Simplex)).join ' '
+    p[:duplex] = sizes.include? "Duplex"
+  end
+  csv
```

Comment:
@malmers does this actually work?

`csv.map` does not mutate `csv`.

Reply:
`each` should probably be used instead of `map`.

`p[:size]` = and `p[:duplex] =` is the reason why the data is updated